functions/src/firebase/env.ts

```diff
@@ -13,11 +13,20 @@ export const getJWTEnv = (): JWTKey => {
 
 export const getTwilioEnv = (): TwilioEnv => {
     const { twilio } = functions.config()
-    if (!twilio.sid || !twilio.authtoken) {
-        throw new InternalServerError('Missing Twilio sid or authtoken')
+    if (
+        !twilio.sid ||
+        !twilio.authtoken ||
+        !twilio.apikeysid ||
+        !twilio.apikeysecret
+    ) {
+        throw new InternalServerError(
+            'Missing Twilio sid or authtoken or apikeysid or apikeysecret'
+        )
     }
     return {
         sid: twilio.sid,
         authToken: twilio.authtoken,
+        apiKeySid: twilio.apikeysid,
+        apiKeySecret: twilio.apikeysecret,
```

Comment:
why the case lack of consistency ?

Reply:
all env are lower case in instantvisio, I've followed this rules since the beginning, I don't have any clue why I choosed this but it's kind of impact full if we change this.